spec/controllers/catalog_controller_spec.rb

```diff
@@ -264,6 +264,49 @@
     end
   end
 
+  describe 'parameters' do
+    describe 'checksums_validated' do
+      let(:results) { instance_double(AuditResults) }
+      let(:poh) { instance_double(PreservedObjectHandler) }
+
+      before do
+        allow(results).to receive(:contains_result_code?)
+        allow(poh).to receive(:results).and_return(results)
+        allow(PreservedObjectHandler).to receive(:new).and_return(poh)
+      end
+
+      it 'false if not present' do
+        expect(poh).to receive(:create).with(false)
+        post :create, params: { druid: bare_druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
+      end
+      ['true', 'True', 'TRUE'].each do |t_val|
+        it "#{t_val} evaluates to true" do
+          expect(poh).to receive(:create).with(true)
+          post :create, params: { druid: bare_druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param, checksums_validated: t_val }
+        end
+      end
+      ['nil', '1', 'on'].each do |t_val|
+        it "#{t_val} evaluates to false" do
+          expect(poh).to receive(:update_version).with(false)
+          patch :update, params: { druid: bare_druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param, checksums_validated: 'false' }
```

Comment:
i think you meant to pass `t_val` instead of `'false'` for `checksums_validated`?  although `'false'`and/or `'FALSE'` might be nice things to add to the list of values being tested.

Reply:
whoops!  good 👀 .   pushed up a fix commit.